doc/whats_new/v0.21.rst

```diff
@@ -393,6 +393,12 @@ Support for Python 3.4 and below has been officially dropped.
   :issue:`12334` by :user:`Emmanuel Arias <eamanu@eamanu.com>`,
   `Joel Nothman`_ and `Andreas Müller`_
 
+- |Enhancement| Added `beta` parameter to
+  :func:`metrics.cluster.supervised.homogeneity_completeness_v_measure` and
+  :func:`metrics.cluster.supervised.v_measure_score` to configure the tradeoff between homogeneity and completeness.
```

Comment:
Please split this line. We try to keep lines under 80 chars

Reply:
My bad ! Changed it now